main.go

```diff
@@ -216,6 +205,14 @@ func main() {
 		*targetFolder = *sourceFolder
 	}
 	*sourceFolder = errors.Must(filepath.Abs(*sourceFolder)).(string)
+	if _, err := os.Stat(*sourceFolder); os.IsNotExist(err) {
+		if !*sourceIgnoreMissing {
+			errors.Printf("Source folder: %s does not exist", *sourceFolder)
+			return 1
+		}
+		return 0
```

Comment:
I would be in favor of logging an informational message here. I know it's the normal case on passing `--ignore-missing-source`, but I can easily imagine debugging and not knowing what's going on in that case.